server/controllers/meeting_controller.coffee

```diff
@@ -45,8 +49,11 @@ meetingController =
           index = emails.indexOf email
           emails.splice(index, 1)
       Meeting.methods.update(meeting_id, {emails:emails})
-      response.schedule = dummy_response
-      res.status(200).send response
+      buildMeetingCalendar emails, (users, availability)->
```

Comment:
@sthoma We should make another story about sending back the meeting initiator's schedule with this data